tools/tf2openapi/README.md

```diff
@@ -14,5 +14,23 @@ The outcome of this project is a TensorFlow-to-OpenAPI transformer which takes S
 ## Caveats
 * There is a dependency on protobufs defined by TensorFlow, e.g. [tensorflow/core/protobuf](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/core/protobuf). Specific protos must be compiled into Go using [protoc](https://github.com/golang/protobuf/tree/master/protoc-gen-go) in the order: tensorflow/core/lib/core/\*.proto, tensorflow/core/framework/\*.proto, tensorflow/core/protobuf/saver.proto, tensorflow/core/protobuf/meta_graph.proto, tensorflow/core/protobuf/saved_model.proto. See Makefile which will automate this.  
 
-  ## TensorFlow Compatibility
-* This tool is compatible with TensorFlow versions 1.xx up to and including 1.13.1. To make it compatible with future TensorFlow versions, you will need to compile the TensorFlow protos and convert them to the internal models. See [DEVELOPER_GUIDE](DEVELOPER_GUIDE.md) for potential issues and solutions.
\ No newline at end of file
+## TensorFlow Compatibility
+* This tool is compatible with TensorFlow versions 1.xx up to and including 1.13.1. To make it compatible with future TensorFlow versions, you will need to compile the TensorFlow protos and convert them to the internal models. See [DEVELOPER_GUIDE](DEVELOPER_GUIDE.md) for potential issues and solutions.
+
+## Usage
+```
+Usage:
+  tf2oas [flags]
+
+Required Flags:
+  -m, --model string           Absolute path of SavedModel file
+  
+Flags:
+  -h, --help                   help for tf2oas
+  -m, --model string           Absolute path of SavedModel file
```

Comment:
--model_base_dir or whatever tensorflow uses?

Reply:
got it, changing to model_base_path https://www.tensorflow.org/tfx/serving/docker#running_a_serving_image